tonic/src/client/grpc.rs

```diff
@@ -296,6 +316,7 @@ impl<T: Clone> Clone for Grpc<T> {
     fn clone(&self) -> Self {
         Self {
             inner: self.inner.clone(),
+            origin: self.origin.clone(),
```

Comment:
While I cannot judge whether this clone or the `ServiceBuilder`’s clone would be more efficient, this tradeoff with the scheme and authority being owned by the `Grpc` seems right to optimize for the expected use cases, making it clear how to replace Channels going forward without needing to learn Tower as well just to get started.